src/editors/sharedComponents/SelectionModal/SearchSort.jsx

```diff
@@ -54,9 +55,18 @@ export const SearchSort = ({
       </Form.Group>
 
       { !showSwitch && <ActionRow.Spacer /> }
-      <SelectMenu variant="link">
+      <SelectMenu variant="link" className="search-sort-menu">
         {Object.keys(sortKeys).map(key => (
-          <MenuItem key={key} onClick={onSortClick(key)} defaultSelected={key === sortBy}>
+          <MenuItem
+            key={key}
+            onClick={onSortClick(key)}
+            defaultSelected={key === sortBy}
+            iconAfter={(key === sortBy) ? Check : null}
+          >
+            <span className="search-sort-menu-by">
+              <FormattedMessage {...messages.sortBy} />
+              &nbsp;
```

Comment:
Instead of using this can we create a space using padding?

Reply:
Using padding makes the underline on hover disconnected:

![image](https://github.com/openedx/frontend-lib-content-components/assets/1616648/c6c10cf0-d3e0-4566-ad8c-33a4f0e83611)

However I was able to use an additional span with white-space: pre-wrap instead of nbsp to achieve the correct appearance, as per the latest commit.